ironic_tempest_plugin/tests/scenario/ironic_standalone/test_basic_ops.py

```diff
@@ -162,6 +162,48 @@ def test_ip_access_to_server(self):
         self.boot_and_verify_node()
 
 
+class BaremetalIdracDriverDirect(bsm.BaremetalStandaloneScenarioTest):
+
+    api_microversion = '1.31'  # to set the deploy_interface
```

Comment:
check api-mcroversion. Should be mapped with aligned functionality ie node deployment